src/main/resources/assets/computercraft/lua/rom/programs/echo.lua

```diff
@@ -0,0 +1 @@
+print( ... )
```

Comment:
This file probably shouldn't be here - it's useful for testing, but has been rejected before (#209).